### PR TITLE
Jxu issue 6 user action

### PR DESCRIPTION
**Pull request by eytoss**
#6

Fix syntax error and introduce hand scoring api structure.

### CODE REVIEW

File: poker/apis.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+hand_score_endpoint = "http://www.pokerbrain.net:88/hand/score"
+
+def score_hand(hand_str):
+    """
+    input a hand string which represents 5 ~ 7 cards
+    return the json obj which
+    contains score and related info for it.
+    """
+    # http://www.pokerbrain.net:88/hand/score?h=d7|d13|d12|d10|s8|d2|h14
+    request_url = "{}?{}".format(hand_score_endpoint, hand_str)
```
Comment (jinpark) [inline poker/apis.py]: you can use params but this is fine :)

```
>>> payload = {'key1': 'value1', 'key2': ['value2', 'value3']}

>>> r = requests.get('http://httpbin.org/get', params=payload)
>>> print(r.url)
http://httpbin.org/get?key1=value1&key2=value2&key2=value3
```

File: poker/models.py [hunk 2]
```diff
@@ -230,7 +221,9 @@ def move_to_next_stage_if_ready(self):
         if self.stage == GameStages.RiverDone:
             # River card has been served and the betting round is done.
             # time for scoring!
-            # TODO: need behavior design
+            # TODO: waiting to use advanced api Jon is working one
+            # score each hand and return the winner
+            score_hand("asdfasdf")
```
Comment (jinpark) [inline poker/models.py]: :+1: :D

**Comment by jinpark** [comment]
LGTM

File: poker/apis.py (hunk 1 above)
Comment (eytoss) [inline poker/apis.py]: that's more clear,  will use when using advanced api.

**Pull request merged**